plugins/logs/plugin.go

```diff
@@ -277,6 +278,21 @@ func (c *Config) validateAndInjectDefaults(services []string, plugins []string)
 		return fmt.Errorf("invalid decision_log config, must have a `service`, `plugin`, or `console` logging enabled")
 	}
 
+	if c.Reporting.Trigger == nil {
+		c.Reporting.Trigger = trigger
+	} else {
+		if *c.Reporting.Trigger != *trigger {
+			return fmt.Errorf("invalid decision_log config, trigger mode mismatch")
```

Comment:
[nit] same as with bundles, we could include the mismatching values

Reply:
Updated the error message here and in the status plugin.